src/xrScriptEngine/ScriptEngineScript.cpp

```diff
@@ -32,7 +32,7 @@ void ErrorLog(pcstr caMessage)
     if (GEnv.ScriptEngine->debugger())
         GEnv.ScriptEngine->debugger()->Write(caMessage);
 #endif
-#ifdef DEBUG
+#ifdef USE_DEBUGGER
```

Comment:
```suggestion
#ifdef USE_LUA_STUDIO
```

Reply:
> > Master Gold is not intended to be enabled in Debug, it's only for Release.
> 
> But! Changes in xrScriptEngine are useful! Though, `USE_LUA_STUDIO` should be used there. Can you submit new PR or reopen this with only changes to xrScriptEngine? :)

Can do. Re-examining the file it looks like `#if defined(USE_DEBUGGER) && defined(USE_LUA_STUDIO)` is used. I will keep in line with that.

Can you reopen the PR and I will push fixed commits?

Reply:
Thanks @Xottab-DUTY. Should be good to go now.